admin_app_2/src/components/LanguageButtonBar.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { Layout } from "@/components/Layout";
+import {
+  DEFAULT_LANGUAGE,
+  LANGUAGE_OPTIONS,
+  appColors,
+  appStyles,
+  sizes,
+} from "@/utils";
+import { Menu, MenuItem, ToggleButton, ToggleButtonGroup } from "@mui/material";
+import { AddCircle, Delete, InfoOutlined } from "@mui/icons-material";
+
+const LanguageButtonBar = ({ expandable }: { expandable: boolean }) => {
+  const [langList, setLangList] = React.useState(
+    expandable
+      ? [LANGUAGE_OPTIONS.find((l) => l.code === DEFAULT_LANGUAGE)]
+      : LANGUAGE_OPTIONS
+  );
+  const [selectedLang, setSelectedLang] = React.useState<string>("en");
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  return (
+    <Layout.FlexBox
+      flexDirection={"row"}
+      gap={sizes.smallGap}
+      {...appStyles.alignItemsCenter}
+    >
+      <ToggleButtonGroup>
+        {langList.map((lang, index) => (
+          <ToggleButton
+            key={lang?.code}
+            value={lang?.code}
+            size="medium"
+            sx={[
+              {
+                border: 0,
+                borderBottomColor: appColors.outline,
+                borderBottomWidth: 1,
+              },
+              selectedLang === lang?.code && {
+                borderBottomColor: appColors.primary,
+                borderBottomWidth: 3,
+                color: appColors.primary,
+              },
+            ]}
+            onClick={() => setSelectedLang(lang?.code)}
+          >
+            {expandable && lang?.code !== DEFAULT_LANGUAGE && (
+              <Delete
+                fontSize="inherit"
+                color="disabled"
+                onClick={() => {
+                  if (
+                    window.confirm(
+                      "Are you sure you want to delete this language?"
+                    )
+                  ) {
+                    setLangList(langList.filter((l) => l !== lang));
+                  }
+                }}
+              />
+            )}
```

Comment:
Should this button and logic be inside on the edit page instead? It's too cramped up here for this also accidental clicks are likely.